README.md

```diff
@@ -41,10 +43,47 @@ cmake .. -DCMAKE_BUILD_TYPE=Release # use Debug for debug build
 make
 ```
 
-## Build knowledge base (from sc-machine/kb folder):
+## Build knowledge base
+
+This repo provides *build_kb.py* script to prepare and build knowledge base (KB). Note that you will need to run a server with a path to a compiled KB in the next step.
+
+### Usage example
 ```sh
-cd sc-machine/scripts
-./build_kb.sh
+cd sc-machine
+python3 scripts/build_kb.py ../ims.ostis.kb -c config/sc-machine.ini
 ```
 
+Default config file saves compiled KB one level above sc-machine, `sc-machine/..`
+
+Alternatively, you can use `build_kb.py` without a config file to define output path and other settings manually:
+```sh
+cd sc-machine
+build_kb.py [-o OUTPUT_PATH] [-l ERRORS_FILE_PATH] [-f REPO_FILE_NAME] repo_folder
```

Comment:
The FILE_NAME is needed to indicate the default name for the repo file (currently `repo.path`). It's used not only for the root repo, but also for all the dependencies of the KB. We were asked to leave this argument as-is because some day repo filename could change in a KB. 

`repo_folder`, on the other hand, is the location of the root repo file, so basically it's not used to specify what's the filename, but where to search for it. We could extract the filename from the path to the repo file and use it in all dependent folders, but then again, I was asked to leave a separate flag for the filename. Did that change?

Reply:
After discussion we made decision to leave one arg: REPO_FILE